.github/workflows/standalone.yml

```
@@ -0,0 +1,75 @@
+name: Build standalone
+
+on:
+  push:
+    branches:
+    - main
+    - 'v*'
+    tags:
+    - 'v*'
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-standalone
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash {0}
+
+
+jobs:
+  build_binary:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os: [ubuntu, windows, macos]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+
+    - name: Install jdaviz
+      run: pip install .
+
+    - name: Install pyinstaller
+      run: pip install pyinstaller
+
+    - name: Create standalone binary
+      run: (cd standalone; pyinstaller ./jdaviz.spec)
```

```suggestion
      run: (cd standalone; pyinstaller -m ./jdaviz.spec)
```
Cami found the executable on macos was not flagged as an executable, requiring a `chmod +x`. This article suggests the manifest flag should fix this: https://jira.stsci.edu/browse/JDAT-3212

---

@duytnguyendtn it looks like you linked the wrong thing here, your comment points to a Jira ticket rather than an article. Any chance you have the original link?

---

FYI this does not work as suggested, the `-m` argument requires a file as input.